api/src/main/java/org/pmiops/workbench/cohortbuilder/FieldSetQueryBuilder.java

```diff
@@ -0,0 +1,320 @@
+              columnFilter.getColumnName());
+    }
+    sqlBuilder.append(columnFilter.getColumnName());
+    sqlBuilder.append(' ');
+    sqlBuilder.append(columnFilter.getOperator());
+    sqlBuilder.append(" ${");
+    sqlBuilder.append(paramName);
+    sqlBuilder.append("}\n");
+  }
+
+  private void handleInClause(ColumnFilter columnFilter, ColumnConfig columnConfig,
+      StringBuilder sqlBuilder, Map<String, QueryParameterValue> paramMap) {
+    String paramName = "p" + paramMap.size();
+    if (columnFilter.getValue() != null || columnFilter.getValueNumber() != null
+        || columnFilter.getValueDate() != null || columnFilter.getValueNull() != null) {
+      throw new BadRequestException("Can't use in operator with single value filter");
+    }
+    List<BigDecimal> valueNumbers = columnFilter.getValueNumbers();
+    List<String> valueStrings = columnFilter.getValues();
+
+    if (valueNumbers == null || valueNumbers.isEmpty()) {
+      if (valueStrings == null || valueStrings.isEmpty()) {
+        throw new BadRequestException("Can't use in operator without values or valueNumbers");
+      } else {
+        if (!columnConfig.type.equals(ColumnType.STRING)) {
+          throw new BadRequestException("Can't use values with column " + columnConfig.name
+              + " of type " + columnConfig.type);
+        }
+        paramMap.put(paramName, QueryParameterValue.array(valueStrings.toArray(new String[0]),
+            String.class));
+      }
+    } else {
+      if (valueStrings == null || valueStrings.isEmpty()) {
+        if (!columnConfig.type.equals(ColumnType.INTEGER)) {
+          throw new BadRequestException("Can't use valueNumbers with column " + columnConfig.name
+              + " of type " + columnConfig.type);
+        }
+        paramMap.put(paramName, QueryParameterValue.array(
+            valueNumbers.stream().map(BigDecimal::longValue).collect(
+                Collectors.toList()).toArray(new Long[0]),
+            Long.class));
+      } else {
+        throw new BadRequestException("Can't use both values and valueNumbers");
+      }
+    }
+    sqlBuilder.append(columnFilter.getColumnName());
+    sqlBuilder.append(" in unnest(${");
+    sqlBuilder.append(paramName);
+    sqlBuilder.append("})\n");
+
+  }
+
+  private void handleColumnFilter(ColumnFilter columnFilter, TableConfig tableConfig,
+      StringBuilder sqlBuilder, Map<String, QueryParameterValue> paramMap) {
+    if (Strings.isNullOrEmpty(columnFilter.getColumnName())) {
+      throw new BadRequestException("Missing column name for column filter");
+    }
+    ColumnConfig columnConfig = findColumn(tableConfig, columnFilter.getColumnName());
+    if (columnConfig == null) {
+      throw new BadRequestException("Column " + columnFilter.getColumnName() + " does not exist");
+    }
+    Operator operator = columnFilter.getOperator();
+    if (operator == null) {
+      operator = Operator.EQUAL;
+    }
```

Comment:
Can you just default columnFilter operator to Equal and remove this null check?

Reply:
The column filters are specified in Swagger. I'm not sure how to specify a default enum value there?

Reply:
I think you can do something like this:
operator:
        $ref: "#/definitions/Operator"
        default: '='

At least in theory this should work. Seems like we are using it for boolean types: 
https://github.com/all-of-us/workbench/blob/master/api/src/main/resources/workbench.yaml#L1480